tools/release.sh

```diff
@@ -42,27 +44,27 @@ shift $((OPTIND-1))
 echo "# Selecting GPG key ..."
 
 gpgkey=$(gpg --list-secret-keys --keyid-format SHORT | awk -F'( +|/)' '/^(sec|ssb)/{print $3}')
-keycount=$(echo $gpgkey | wc -w)
+keycount=$(echo "$gpgkey" | wc -w)
 
-if [ $keycount -eq 0 ]; then
-  echo 'Need at least one GPG key, please make one with `gpg --gen-key`'
-  echo 'You will also need to submit your key to a public keyserver, e.g.'
-  echo '  https://sks-keyservers.net/i/#submit'
+if [ "$keycount" -eq 0 ]; then
+  echo "Need at least one GPG key, please make one with `gpg --gen-key`"
+  echo "You will also need to submit your key to a public keyserver, e.g."
+  echo "  https://sks-keyservers.net/i/#submit"
```

Comment:
These should probably remain as single‑quoted strings, given that, unlike JS, single‑quoted strings are treated as raw strings by **POSIX**‑compatible shells.

Reply:
Yeah there is no need to change full quoting to partial quoting here.